packages/fast-components-react-msft/src/button/button.spec.tsx

```diff
@@ -186,14 +128,13 @@ describe("button", (): void => {
         };
 
         const rendered: any = mount(
-            <Button {...props} />
+            <MSFTButton {...props} />
```

Comment:
Just curious why these are MSFT buttons?

Reply:
@nalogiudice I think this was testing the unstyled vs styled instance (of the MSFT package button).

Reply:
Test doesn't have anything to do with the managed component. We're testing the capabilities of the Microsoft button, not a *styled* Microsoft button.